packages/dumb/src/components/stats/total-carbs/total-carbs-stat.tsx

```diff
@@ -28,73 +28,42 @@ import React, { type FunctionComponent, memo } from 'react'
 
-export interface TotalInsulinStatProps {
-  foodCarbsPerDay: number
-  totalCarbsPerDay: number
-  totalEntriesCarbWithRescueCarbs: number
+export interface TotalCarbsStatProps {
+  title: string
+  annotation?: string[]
+  value: number
 }
-
-const TotalCarbsStat: FunctionComponent<TotalInsulinStatProps> = (props) => {
-  const { totalCarbsPerDay, foodCarbsPerDay, totalEntriesCarbWithRescueCarbs } = props
-  const { t } = useTranslation('main')
-  const location = useLocation()
-  const isDailyPage = location.pathname.includes('daily')
-  const isDerivedCarbs = foodCarbsPerDay && totalCarbsPerDay ? t('tooltip-total-derived-carbs', { total: totalEntriesCarbWithRescueCarbs }) : t('tooltip-empty-stat')
-
+const TotalCarbsStat: FunctionComponent<TotalCarbsStatProps> = (props) => {
+  const { annotation, value, title } = props
   return (
-    <div data-testid="total-carbs-stat">
-      <Box className={styles.row}>
-        {t(isDailyPage ? 'total-carbs' : 'avg-daily-carbs')}
+    <Box className={`${styles.row} ${!annotation ? styles.carbs : null}`}>
+      {title}
+      {annotation &&
         <StatTooltip
-          annotations={[t(isDailyPage ? 'tooltip-total-day-carbs' : 'tooltip-total-week-carbs'), isDerivedCarbs]}
+          annotations={annotation}
         />
-        {!totalCarbsPerDay
-          ? <>
-            <div className={styles['disabled-line']} />
-            <Box className={styles['disabled-label']} fontSize="24px" marginLeft="auto">
-              --
-            </Box>
-          </>
-          : <>
-            <div className={styles.total}>
+      }
+      {!value
+        ? <>
+          <div className={styles['disabled-line']}/>
+          <Box className={styles['disabled-label']} fontSize="24px" marginLeft="auto">
+            --
+          </Box>
+        </>
+        : <>
+          <div className={styles.total}>
                 <span className={styles.value}>
-                  {totalCarbsPerDay}
+                  {value}
                 </span>
-              <span className={styles.suffix}>
+            <span className={styles.suffix}>
                   {Unit.Gram}
                 </span>
-            </div>
-          </>
-        }
-      </Box>
-
-      <Box className={`${styles.rescueCarb} ${styles.row}`}>
-        {t('Rescuecarbs')}
-        {!foodCarbsPerDay
-          ? <>
-            <div className={styles['disabled-line']} />
-            <Box className={styles['disabled-label']} fontSize="24px" marginLeft="auto">
-              --
-            </Box>
-          </>
-          : <>
-            <div className={styles.total}>
-              <span className={styles.value}>
-                {foodCarbsPerDay}
-              </span>
-              <span className={styles.suffix}>
-                {Unit.Gram}
-              </span>
-            </div>
-          </>
-        }
-      </Box>
-    </div>
+          </div>
+        </>
+      }
+    </Box>
   )
 }
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Change to revert, it's a good thing to add blank lines between statements to improve code readability (if it was an automated change, check your IDE settings)